packages/core/contracts/polygon/README.md

```diff
@@ -37,6 +37,6 @@ Any message can be relayed between Polygon and Ethereum provided that they are s
 
 # Security Considerations
 
-This system relies on the [Polygon consensus mechanism](https://docs.matic.network/docs/home/architecture/security-models#proof-of-stake-security) secured by validators in a Proof of Stake system. The validator set enforces the integrity of data passed between networks (i.e. downstream users need to trust that the validators are not modifying the arbitrary messages that are beign sent between networks).
+This system relies on the [Polygon consensus mechanism](https://docs.matic.network/docs/home/architecture/security-models#proof-of-stake-security) secured by validators in a Proof of Stake system. The validator set enforces the integrity of data passed between networks (i.e. downstream users need to trust that the validators are not modifying the arbitrary messages that are being sent between networks).
```

Comment:
The PR also introduces a README file with a brief overview of the Polygon Tunnel architecture. Our comments are:

- On [line 42](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/polygon/README.md#L42) "timeley" should be "timely".
- On [line 40](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/polygon/README.md#L40) "beign" should be "being".